app/views/finance/provider/shared/_line_item.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= content_tag_for :tr, line_item, :class => "line_item" do %>
+<%= content_tag :tr, id: dom_id(line_item), class: "line_item" do %>
```

Comment:
```suggestion
<tr id=dom_id(line_item) class="line_item">
```

Reply:
This is wrong, because it will be rendered as `<tr id="dom_id(line_item)" class="line_item">`, and then the deletion (that depends on JS and element IDs) doesn't work.

The right way would be `<tr id="<%= dom_id(line_item) %>" class="line_item">`, but this looked ugly to me, so in the cases where we needed to keep the ID, I opted for `content_tag`, which seems cleaner.

Reply:
I searched through the code base, and we use `content_tag` a lot in the templates, so it seemed consistent to me.